.projenrc.ts

```diff
@@ -101,6 +101,7 @@ const project = new CustomTypescriptProject({
     "graphql-request",
     "ts-node",
     "ts-patch",
+    "flatted",
```

Comment:
Since I wanted to thes the automatically generated role policy for step functions I had to update the `normalizeCDKJson` util to handle circular objects: https://github.com/functionless/functionless/pull/414/files#diff-8951d3f508f54ec05c5df49f04f821a2aec5c75bce858639461f388e498a398cR11